DfE.FindInformationAcademiesTrusts/Program.cs

```diff
@@ -168,21 +172,24 @@ private static void AddEnvironmentVariablesTo(WebApplicationBuilder builder)
             .Bind(builder.Configuration.GetSection(AcademiesApiOptions.ConfigurationSection))
             .ValidateDataAnnotations()
             .ValidateOnStart();
+
+         builder.Services.AddOptions<TestOverrideOptions>()
+            .Bind(builder.Configuration.GetSection(TestOverrideOptions.ConfigurationSection));
     }
 
     private static void AddAuthenticationServices(WebApplicationBuilder builder)
     {
-        if (ShouldSkipAuthentication(builder))
-            return;
-
         builder.Services.AddAuthorization(options =>
         {
             var policyBuilder = new AuthorizationPolicyBuilder();
             policyBuilder.RequireAuthenticatedUser();
             options.DefaultPolicy = policyBuilder.Build();
             options.FallbackPolicy = options.DefaultPolicy;
         });
+
+         if (!builder.Environment.IsContinuousIntegration())
```

Comment:
Looks like the formatting of this file is a little off - can we run it through Rider's cleanup tool?

Reply:
fixed